app/presenters/setting_presenter.rb

```diff
@@ -15,20 +15,24 @@ class SettingPresenter
   attribute :settings_type, :string
   attribute :select_values
   attribute :config_file
+  attribute :updated_at, :datetime
+  attribute :created_at
   attr_accessor :options
 
   def self.from_setting(setting)
-    SettingPresenter.new({:id => setting.id,
-                          :name => setting.name,
-                          :category => setting.category,
-                          :description => setting.description,
-                          :settings_type => setting.settings_type,
-                          :default => setting.default,
-                          :full_name => setting.full_name,
-                          :config_file => setting.class.config_file,
-                          :select_values => setting.select_collection,
-                          :value => setting.value,
-                          :encrypted => setting.encrypted? })
+    SettingPresenter.new({id: setting.id,
+                          name: setting.name,
+                          category: setting.category,
+                          description: setting.description,
+                          settings_type: setting.settings_type,
+                          default: setting.default,
+                          full_name: setting.full_name,
+                          updated_at: setting.updated_at,
+                          created_at: setting.created_at,
+                          config_file: setting.class.config_file,
+                          select_values: setting.select_collection,
+                          value: setting.value,
+                          encrypted: setting.encrypted? })
```

Comment:
Formatting changes in the same commit make it much harder to review.

Reply:
You're right... my bad, I've separated it into two commits.